sparse_ho/optimizers/line_search.py

```diff
@@ -115,9 +115,10 @@ def _grad_search(
             value_outer_old = value_outer
 
             if self.verbose:
-                print('value outer', value_outer)
-                # print('grad outer', grad_outer)
-                # print('value of log_alphak', log_alphak)
+                print(
+                    "Iteration %i / %i ||" % (i+1, self.n_outer) +
+                    "Value outer criterion: %f ||" % value_outer +
+                    "norm grad %f" % norm(grad_outer))
```

Comment:
you should use f-strings these days

Reply:
also remove spaces around / after Iteration
norm grad can be in scientific mode with 2 significant digits maybe ? %.2e